protected/humhub/components/console/UrlManager.php

```diff
@@ -29,7 +29,7 @@ public function init()
             $hostInfo .= ':' . $urlParts['port'];
         }
         $this->setHostInfo($hostInfo);
-        $this->setScriptUrl($this->getBaseUrl() . ($this->getScriptUrl() ?: '/index.php'));
+        $this->setScriptUrl($this->getBaseUrl() . (str_replace($this->getBaseUrl(), '', $this->getScriptUrl()) ?: '/index.php'));
```

Comment:
As a hotfix for the `master` branch, this change is fine.

But we should create a second PR and refactor this code.

According to the Yii2 documentation, the scriptUrl should contain the relative URL.
https://www.yiiframework.com/doc/api/2.0/yii-web-request#setScriptUrl()-detail

We should therefore not force the BaseURL in front of it.

Can you create another PR for `develop` here and this one for `master`?